content/pop-basics.md

```diff
@@ -17,30 +17,169 @@ section: pop
 tableOfContents: true
 ---
 
+Welcome to the Pop!_OS desktop! By default, it's clean and ready for action.
 The Pop!_OS operating system provides all the features you would expect from a modern desktop.
 
-## Getting to Know The Pop!_OS Desktop
+Pop!\_OS is based on Ubuntu, however, extra features and tools have been added, while other aspects have been slimmed down. You can read more about Pop!\_OS features and the differences between Pop!\_OS and Ubuntu [here](/articles/difference-between-pop-ubuntu)
```

Comment:
Missing a period.

Reply:
Added. This was a previously recommended style choice. I'm still trying to break that habit.